src/bin/util/test/p11prov-test.sh

```
@@ -0,0 +1,189 @@
+
+
+# get absolute path to SoftHSM pkcs#11 module
+# NOTE: Depending on the build model, the module
+# may be located in a subdirectory.
+D=`cd ../../../lib/ && pwd`
+if test -z "$D" ; then
+  echo "unexpectedly missing library directory" >&2
+  exit 99
+fi
+P11MODULE=
+for F in `find "$D" -name '*softhsm2.*'` ; do
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)